third_party/2and3/requests/api.pyi

```diff
@@ -20,7 +20,7 @@ def get(url: Union[Text, bytes],
         **kwargs) -> Response: ...
 def options(url: Union[str, Text], **kwargs) -> Response: ...
 def head(url: Union[str, Text], **kwargs) -> Response: ...
-def post(url: Union[str, Text], data: _Data = ..., json: Optional[MutableMapping] = ..., **kwargs) -> Response: ...
+def post(url: Union[str, Text], data: _Data = ..., **kwargs) -> Response: ...
```

Comment:
I think that even if it's optional, we should still list "json" explicitly. (As `json = ...`)